src/config/config.cc

```diff
@@ -186,10 +186,12 @@ Config::Config() {
       {"rocksdb.enable_pipelined_write", true, new YesNoField(&rocks_db.enable_pipelined_write, false)},
       {"rocksdb.stats_dump_period_sec", false, new IntField(&rocks_db.stats_dump_period_sec, 0, 0, INT_MAX)},
       {"rocksdb.cache_index_and_filter_blocks", true, new YesNoField(&rocks_db.cache_index_and_filter_blocks, false)},
+      {"rocksdb.block_cache_size", true, new IntField(&rocks_db.block_cache_size, 4096, 0, INT_MAX)},
+      {"rocksdb.set_block_cache_size", true, new YesNoField(&rocks_db.set_block_cache_size, true)},
```

Comment:
We can remove this configuration and set the `block_cache_size` default value to 0 in the previous line, then can check if `block_cache_size` was enabled or not.